src/screenSharing/ScreenSharingRequestTracker.ts

```
@@ -107,7 +126,7 @@ export class ScreenSharingRequestTracker {
           return;
         }
 
-        cb({ video: selectedSource });
+        cb(shareAudio ? ({ video: selectedSource, audio: 'loopback' } as any) : { video: selectedSource });
```

ESLint

[error] Replace `shareAudio·?·({·video:·selectedSource,·audio:·'loopback'·}·as·any)·:·{·video:·selectedSource·}` with `⏎··········shareAudio⏎············?·({·video:·selectedSource,·audio:·'loopback'·}·as·any)⏎············:·{·video:·selectedSource·}⏎········`

(prettier/prettier)